source/numerics/error_estimator.cc

```diff
@@ -464,6 +464,98 @@ namespace internal
       return face_integral;
     }
 
+    /**
+     * A factor to scale the integral for the face at the boundary.
+     * Used for Neumann BC (?).
```

Comment:
i was not sure about this comment initially, but that's what indeed happens, i.e. `integrate_over_regular_face()` for `face->at_boundary()` is called for faces in Neumann part of the boundary.